adoc/admin-monitoring-stack.adoc

```diff
@@ -448,6 +448,64 @@ helm upgrade prometheus suse/prometheus --namespace monitoring --values promethe
 * **External IPs**: `+https://prometheus-alertmanager.example.com+`
 * **LoadBalancer**: `+https://prometheus-alertmanager.example.com+`
 
+[[recording_rules_configuration_example]]
+==== Recording Rules Configuration Example
+
+Recording rules allow you to precompute frequently needed or computationally
+expensive expressions and save their result as a new set of time series.
+Querying the precomputed result will then often be much faster than executing
+the original expression every time it is needed. This is especially useful for
+dashboards, which need to query the same expression repeatedly every time they
+refresh. Another common use case is federation where precomputed metrics are
+scraped from one Prometheus instance by another.
+
+For more information on how to configure recording rules, refer to
+link:https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#recording-rules[Prometheus:Recording Rules - Configuration].
+
+. Configuring recording rules
++
+Add the following group of rules in the `serverFiles` section of the configuration file.
```

Comment:
My only note would be that saying "the configuration file" is a little vague here, but the 'helm upgrade' command below gives a file name below, so I was able to figure it out. :)